Update index.md of 'Printing output' in NodeJs Roadmap (#5287)
kamranahmedse/developer-roadmap: src/data/roadmaps/nodejs/content/106-nodejs-command-line-apps/101-printing-output/index.md

# Printing output

> Printing output in Node.js command-line applications is a fundamental aspect of communicating with users and displaying information. Here's how you can print output in Node.js command-line apps:
> 
> - **process.stdout**: This is a writable stream used for printing normal program output.
> - **process.stderr**: This is a writable stream used for printing error messages and warnings.
> - **Chalk**: Chalk is a popular package for styling and adding colors to terminal output. It provides a simple and expressive API for applying styles to strings.
> - **Figlet**: Figlet allows you to create ASCII art from text. It's useful for creating stylized text headings or banners in terminal applications.
> - **cli-progress**: cli-progress is a simple and flexible progress bar package for Node.js command-line applications. It provides options for customizing the appearance and behavior of progress bars.
> 
> Here's a simple example of using **process.stdout** and **process.stderr**:
> 
> ```
> process.stdout.write('This is standard output\n');
> ```
> 
> ```
> process.stderr.write('This is an error message\n');
> ```

Reviewer: ```suggestion

Printing output in Node.js command-line applications is a fundamental aspect of communicating with users and displaying information. You can print output using `process.stdout`, `process.stderr` or simple `console.*` functions.
```
